simulation-runner/src/main/scala/com/puppetlabs/gatling/runner/ConfigDrivenSimulation.scala

```diff
@@ -101,7 +101,10 @@ class ConfigDrivenSimulation extends Simulation {
     val chainWithSleeps:ChainBuilder =
       addSleeps(chainWithFailFast, sleepDuration, numRepetitions)
 
+    val feeder = NodeFeeder("node", numInstances).circular
```

Comment:
`num_instances` from a scenario gets converted into `numInstances` here somewhere along the way?